database/migrations/default/1737481505501_specify-schema-for-jurisdictions/up.sql

```diff
@@ -0,0 +1,78 @@
+drop trigger if exists afd_incidents_trigger_insert on public.afd__incidents;
+
+drop trigger if exists afd_incidents_trigger_update on public.afd__incidents;
+
+
+drop function public.afd_incidents_trigger ();
+
+create or replace function public.afd_incidents_trigger()
+returns trigger
+language plpgsql
+as $function$
+BEGIN
+  WITH jurisdiction_union AS (
+    SELECT ST_Union(geometry) AS geometry
+    FROM geo.jurisdictions
+    WHERE jurisdiction_label = 'AUSTIN FULL PURPOSE'
+  )
+  UPDATE afd__incidents
+  SET
+    austin_full_purpose = (
+      SELECT ST_Contains(jurisdiction_union.geometry, incidents.geometry)
+      FROM afd__incidents incidents, jurisdiction_union
```

Comment:
And using that ephemeral single piece of geometry instead of the table which now lives in `geo.` and no longer in `public.`, which is related to what brought this all to light.